modules/persistent-storage-csi-sc-managing-cli.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-managing-cli_{context}"]
+= Managing the default storage class using the CLI
+
+.Prerequisites
+* Access to the cluster with cluster-admin privileges.
+
+.Procedure
+
+To manage the storage class using the CLI, run the following command:
+
+[source, terminal]
+----
+oc patch clustercsidriver $DRIVERNAME --type=merge -p "{\"spec\":{\"storageClassState\":\"${STATE}\"}}" <1>
+----
+<1> Where `${STATE}` is "Removed" or "Managed" or "Unmanaged".
```

Comment:
Do you think it's worth explaining `$DRIVERNAME` too, or is it obvious enough?

Reply:
I think it would be helpful to say that $DRIVERNAME refers to the CSI driver and list the possible drivers that are supported with this feature. Thoughts?

Reply:
@dobsonj are we supporting all CSI drivers for 4.13?

Reply:
$DRIVERNAME refers to the provisioner name that you can get by doing a `oc get sc`
Driver list is available here https://issues.redhat.com/browse/STOR-947
It applies to drivers managed by CSO not the ones that are under OLM such as AWS EFS or GCP filestore